app/javascript/player/index.jsx

```diff
@@ -63,12 +63,23 @@ class Player extends React.Component<Props, { playing: boolean }> {
     if (url.searchParams.has('autoplay')) {
       this._play()
     }
+
+    var audio = this.audioElement
+
+    const volumeBar = document.getElementById('volume-bar')
```

Comment:
Let’s not circumvent React.

Add a `handleChangeVolume` method to `Player` which changes the volume, and pass a bound reference (use an arrow function) down to song information and use it as the `onChange` handler for the slider.